app/assets/stylesheets/image_x.scss

```diff
@@ -2,3 +2,67 @@
 @import 'modules/media';
 @import 'vendor/tooltip';
 
+.#{$namespace}-image-x {
+  height: 400px;
+  position: relative;
+}
+
+.#{$namespace}-image-x-thumb-slider-container {
+  background: rgba(0, 0, 0, 0.5);
+  border: 1px solid #000000;
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+}
+
+.#{$namespace}-image-x-thumb-slider-open-close {
+  cursor: pointer;
+  text-align: center;
+  width: 100%;
+}
+
+.#{$namespace}-image-x-thumb-slider {
+  height: 140px;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  ul {
+    height: 100%;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.#{$namespace}-image-x-thumb-slider-scroll {
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.